internal/bowtie/client/http_test.go

```diff
@@ -35,6 +35,21 @@ func TestClient_getHostURL(t *testing.T) {
 			},
 			want: "https://example.com/-net/api/v0/user/login",
 		},
+		{
+			name: "local login",
+			fields: fields{
+				HTTPClient: nil,
+				hostURL:    "http://localhost:3000",
+				auth: AuthPayload{
+					Username: "test@example.com",
+					Password: "passw0rd123",
```

Comment:
Does this purposefully fail zxcvbn? 
https://old.formvalidation.io/examples/building-password-strength-meter/

Reply:
All this test is doing is testing that my url building template works for calculating where to send requests. My guess is tyler added this one to just make sure we caught any awkwardness with http and localhost

Reply:
Yeah, that's all I was doing here - ensuring that the login logic was hitting the endpoint I expected.